packages/rulesets/src/asyncapi/index.ts

```diff
@@ -55,6 +57,17 @@ export default {
         },
       },
     },
+    'asyncapi-channel-parameters-defined': {
```

Comment:
I don't know if it's a good name, or maybe simpler `asyncapi-channel-parameters` will be better.

Reply:
👍 IMHO `asyncapi-channel-parameters` is nicer (and more consistent with similar rules in other rulesets).

Reply:
Thanks! Changed :)